src/main/java/de/uni_kl/cs/discodnc/feedforward/arrivalbounds/AggregatePmoo.java

```diff
@@ -39,6 +39,7 @@
 import de.uni_kl.cs.discodnc.bounds.disco.pwaffine.Bound;
 import de.uni_kl.cs.discodnc.curves.ArrivalCurve;
 import de.uni_kl.cs.discodnc.curves.Curve;
```

Comment:
Remove obsolete import

Reply:
I'm on the #43_Create_a_constant_holder branch and checked online, but I cannot find usage of Curve. Can you point me to the line where Curve is used? Maybe I overlooked it.

It makes sense that Curve is not used anymore. Before, Curve was usually only used as Curve.getFactory(), a shortcut for Calculator.getInstance().getCurveFactory(), to then get what is now constants. Additionally, in these classes is no need to add up arrival curves. This is done by ArrivalBoundDispatch (adding up the arrivals from flows originating at a server as well as those coming from a inturn/inlink to this server).